Resources/public/js/models/ez-locationmodel.js

```diff
@@ -381,6 +403,18 @@ YUI.add('ez-locationmodel', function (Y) {
                     return contentInfo;
                 }
             },
+
+            /**
+             * The location's path. By default it is empty. You need to call the loadPath method to set this attribute.
+             *
+             * @attribute path
+             * @default false
+             * @type array
+             * @readOnly
```

Comment:
that's not enough to set an attribute as read only, see http://yuilibrary.com/yui/docs/attribute/#howtoconfig